src/EventListener/SerializeListener.php

```diff
@@ -47,7 +47,7 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
         $controllerResult = $event->getControllerResult();
         $request = $event->getRequest();
 
-        if ($controllerResult instanceof Response) {
+        if ($controllerResult instanceof Response || !$request->attributes->get('_api_respond')) {
```

Comment:
shouldn't we keep the actual default behavior like so :
```suggestion
        if ($controllerResult instanceof Response || !$request->attributes->get('_api_respond', true)) {
```